app/views/import/index.html.erb

```diff
@@ -35,11 +35,11 @@
 
     <h4 class="h4 mt-4 mb-2">Requirements:</h4>
     <ul class="ml-4 mb-4-sm">
-      <li><span class="text-bold">Valid columns:</span> rank, heroes, map, comment, time, day, ally thrower, ally leaver, enemy thrower, enemy leaver, group, result, placement</li>
+      <li><span class="text-bold">Valid columns:</span> rank, heroes, map, comment, time, day, ally thrower, ally leaver, enemy thrower, enemy leaver, group, result, placement, potg</li>
       <li>Rank is required for non-placement matches</li>
       <li>Valid values for 'day' column: weekday, weekend</li>
       <li>Valid values for 'time' column: morning, afternoon, evening, night</li>
-      <li>Valid values for thrower, leaver, and placement columns: Y, N</li>
+      <li>Valid values for thrower, leaver, placement, and potg columns: Y, N</li>
```

Comment:
I would write out “play of the game” here, then maybe put “(PotG)” after to build familiarity if the user exports their matches and sees that abbreviation.